Test.lean

```diff
@@ -0,0 +1,10 @@
+import Mathlib
+import Mathbin
```

Comment:
@semorrison does this file also take a long time for you? At least in the Docker container, it takes 24 seconds:
```
lean@799ae2b21f76:~/mathport$ time make test  
LEAN_PATH=./build/lib:./lean_packages/mathlib/build/lib:./Lib4/leanbin/build/lib:./Lib4/mathbin/build/lib lean ./Test.lean
Semiring : Type u_1 → Type u_1
Semiringₓ
Semiringₓ : Type u_1 → Type u_1
Nat.exists_infinite_primes

real	0m25.611s
user	0m25.242s
sys	0m0.367s
```

Reply:
I get (not the in the Docker container):
```
% time make test
LEAN_PATH=./build/lib:./lean_packages/mathlib/build/lib:./Lib4/leanbin/build/lib:./Lib4/mathbin/build/lib lean ./Test.lean
Semiring : Type u_1 → Type u_1
Semiringₓ
Semiringₓ : Type u_1 → Type u_1
Nat.exists_infinite_primes
make test  14.91s user 0.54s system 99% cpu 15.463 total
```

Reply:
This is pretty much the same as Lean3:
```
% scripts/mk_all.sh 
% time lean --make src/all.lean
lean --make src/all.lean  14.22s user 2.24s system 100% cpu 16.446 total
```